06-building-a-program-start-to-finish/writing-tests/output/tests.js

```diff
@@ -0,0 +1,209 @@
+
+/* eslint-disable no-undef */
+
+const data = require('./data.json');
+const book = require('./books.json');
+const invalid = require('./invalid.json');
+const invalidContent = require('./invalid-content.json');
```

Comment:
A base filename should exactly match the name of its default export. [See link](https://github.com/airbnb/javascript#naming--filename-matches-export)